drupal/code/modules/features/pori_configuration_overrides_feature/pori_configuration_overrides_feature.module

```diff
@@ -5,3 +5,24 @@
  */
 
 include_once 'pori_configuration_overrides_feature.features.inc';
+
+/**
+ * Implements hook_form_alter().
+ */
+function pori_configuration_overrides_feature_form_alter(&$form, &$form_state, $form_id) {
+  global $user;
+
+  $logged_in_user = user_load($user->uid);
```

Comment:
We should first check that we're on the right form either by using the hook_form_FORM_ID_alter (which would be more efficient) or at least by checking the form id first and only then making the user_load(), since it's a pretty expensive operation. The user_load() and/or the following assign line also potentially fails with anonymous users for example on the login form or any other form, which causes either log noise or white page.